### DISCUSSION TITLE
Hide ground item text unless hovering over that tile

### DISCUSSION THREAD

**Discussion by beardhub** [runelite/runelite#14957] [Ideas]
Add an option to only display the text for ground items when hovering your mouse over that tile. They should still have the normal color/highlights to indicate the presence/rarity of items, but I think hiding the text like this would make things look cleaner. 

I believe currently pressing alt or another hotkey can show/hide all ground items in a similar fashion, but this method would let you view specific tiles instead of everything in the area.

It could more or less be an intersection with the Tile Indicators' highlight hovered tile setting

**Comment by EliasLahham** [comment]
Looking into this

**Comment by EliasLahham** [comment]
Work in progress:
![itemHover](https://user-images.githubusercontent.com/17418745/161396065-d2b77679-01b8-4d01-9e8c-fb255cc65049.gif)

I need to figure out how to handle more expensive item drops. I imagine they would all fall under the same category give the user has this new setting checked

edit: [see PR](https://github.com/runelite/runelite/pull/14824)

**Comment by EliasLahham** [comment]
Final product from PR: https://imgur.com/UpVzLgo
